backend/handlers/handlers.go

```diff
@@ -765,11 +770,38 @@ func (handlers *Handlers) postExportAccountSummary(_ *http.Request) (interface{}
 		accountName := account.Name()
 		balance := account.Balance().Available().BigInt().String()
 		unit := account.Coin().SmallestUnit()
+		var accountType string
+		var xpubs string
+		var address string
+		var scriptType string
+		if signingConfiguration := account.Info().SigningConfiguration; signingConfiguration.IsAddressBased() {
+			accountType = "address"
+			address = signingConfiguration.Address()
+			scriptType = ""
+		} else {
+			accountType = "xpub"
+			switch specificAccount := account.(type) {
+			case *btc.Account:
+				for _, xpub := range specificAccount.Info().SigningConfiguration.ExtendedPublicKeys() {
+					xpubs += xpub.String() + "; "
+				}
+				scriptType = string(specificAccount.Info().SigningConfiguration.ScriptType())
```

Comment:
not needed, it's already `""`.

Reply:
You mean the `string(…)` part is not needed?

Reply:
i think i commented on a commit, and when looking a all commits, the comment is on the wrong line...

the comment applies for where `scriptType = ""`